### PR TITLE
Circ 568 Modify End Session endpoint to support list of patronId

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Purpose
To update end session API to support list of patronId. 
Resolves: [CIRC-568](https://issues.folio.org/browse/CIRC-568)

## Approach
- update logic for EndPatronActionSessionResource for using array of objects with patronId and actionType;
- update tests according to logic changes

## Notes
Notes
This PR should be merged together with ui-circulation PR (not ready yet)

## Links
[CIRC-568](https://issues.folio.org/browse/CIRC-568)
[CIRC-412](https://issues.folio.org/browse/CIRC-412)

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] Were any API paths or methods changed, added or removed?
  - [x] Were there any schema changes?
  - [x] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -440,7 +440,7 @@
     },
     {
       "id": "patron-action-session",
-      "version": "0.1",
+      "version": "1.1",
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: This API is marked experimental, why move it to a non-experimental API for this change?

File: ramls/end-patron-action-session-requests.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (marcjohnson-kint) [inline ramls/end-patron-action-session-requests.json]: This JSON schema appears to be new, yet there appears to be no new reference. Were is it referenced?

**Comment by marcjohnson-kint** [comment]
> ERROR: lint-raml-cop:   raml-cop detected errors with end-patron-action-session.raml:
> [ramls/examples/end-patron-action-session-request.json:0:0] ERROR Content is not valid according to schema: Missing required property: actionType
> [ramls/examples/end-patron-action-session-request.json:0:0] ERROR Content is not valid according to schema: Missing required property: patronId

Please fix the reported errors with the JSON examples

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (roman-barannyk) [reply descriptors/ModuleDescriptor-template.json]: updated version

File: ramls/end-patron-action-session-requests.json (hunk 2 above)
Comment (roman-barannyk) [reply ramls/end-patron-action-session-requests.json]: fixed

**Comment by kostyantyn-kh** [comment]
Please fix 
ERROR: lint-raml-cop:   raml-cop detected errors with end-patron-action-session.raml:
[ramls/examples/end-patron-action-session-request.json:0:0] ERROR Content is not valid according to schema: Missing required property: totalRecords

**Comment by roman-barannyk** [comment]
reported errors with the JSON examples have been fixed

**Pull request merged**